src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -73,7 +73,7 @@
     public void getSalaryInfoByMonth() throws Exception {
         try {
             Class<?> exceptionClass = Class
-                    .forName("core.basesyntax.exception.IllegalDateParametersException");
+                    .forName("core.basesyntax.IllegalDateParametersException");
```

Comment:
you are not allowed to rewrite tests. put your custom exception to the `exception` package